session_0/word_processing.rb

```diff
@@ -1,14 +1,23 @@
+# Code By : Akshay Jain
+
 # The function `lower_case` takes an array of strings and converts
 # any non-lower case alphabet (A..Z) to corresponding lower case
 # alphabet
 def lower_case(words)
-  raise NotImplementedError # TODO
+  lower_words = []
```

Comment:
Can we use `map` here? https://www.rubyguides.com/2018/10/ruby-map-method/

Reply:
Yes..for lower_case method we can use map method and for lower_case! method we can use map! method.
That's a great info :+1: 
For lower_case : words.map { |word| word.downcase())
For lower_case! : words.map! { |word| word.downcase())